src/components/dataSet-settings.js

```diff
@@ -370,15 +360,12 @@ class DataSetSettings extends React.Component {
                                             ) {
                                                 const dataSet = this
                                                     .specificSettings[key]
-                                                const sumaryString =
-                                                    dataSet.periodDSDownload
                                                 const sumarySettings =
                                                     dataSet.periodDSDownload ===
                                                     undefined
                                                         ? undefined
-                                                        : this.arrangeWord(
-                                                              sumaryString
-                                                          )
+                                                        : dataSet.periodDSDownload
```

Comment:
This also looks like a functional change?

Reply:
Indeed this is a functional change, the thing is that on my dataStore I was already using the input number so it sent me a render error, I think I'm also removing these lines in another PR, I'll keep the lines.